commons-statistics-distribution/src/main/java/org/apache/commons/statistics/distribution/LogCauchyDistribution.java

```diff
@@ -111,13 +111,10 @@ public double getVariance() {
         return Double.POSITIVE_INFINITY;
     }
 
-    /* The distribution is not defined for x less than or qual to zero */
-
     /**
      * {@inheritDoc}
      *
-     * The lower bound of the support is zero no matter the parameters.
-     * The lower bound is open ended for the Support
+     * The lower bound of the support is always zero no matter the parameters.
      *
      * @return lower bound of the support (always Double.NEGATIVE_INFINITY)
```

Comment:
You also need to change this `(always Double.NEGATIVE_INFINITY)` to `(always zero)`